templates/common/_base/files/sysctl-forward-conf.yaml

```diff
@@ -5,4 +5,5 @@ contents:
   inline: |
     net.ipv4.ip_forward = 1
     net.ipv6.conf.all.forwarding = 1
+    vm.unprivileged_userfaultfd = 1
```

Comment:
Should we rather create a new file for this setting? This seems to be relevant only for  `OpenShiftSDN` which is not default anymore.

Reply:
Done!